🔧-How-to-Install.md

```diff
@@ -1,9 +1,14 @@
 ### 🐳 Docker
 
 ```bash
+curl -sSL https://get.docker.com/ | CHANNEL=stable sh
+apt update && apt install docker-compose-plugin
```

Comment:
You are including a lot of assumption here.
Please link to the approporiate docs instead.
We can not support users having trouble with installing docker.